version.config

```
@@ -1,3 +1,2 @@
 version = 2.20.0-dev
-previous_version = 2.19.2
-downgrade_to_version = 2.19.2
+previous_version = 2.19.3
```

```suggestion
previous_version = 2.19.3
downgrade_to_version = 2.19.3
```
Need to keep this until after 2.20.0 release

---

Ah yes! Forgot that the cut-off is 2.20 release. Thanks for the dilligence